### DISCUSSION TITLE
Web client active session

### DISCUSSION THREAD

**Discussion by mxav1111** [benoitc/gunicorn#3313] [Q&A]
Hi there,
I am trying to automate a certificate renewal and looking for way to avoid user disruption if possible when certificate expires.
Is there a way to identify if web client is/was active in last N (let's 30 or so ) mins? If so, we can wait out a bit before bouncing.
Any thoughts or ideas? Or a way to force renewal without bounce that will be taken up by next web client connection?

**Comment by mxav1111** [comment]
:bump:

**Reply by benoitc** [reply]
maybe you check the lqst time q metric hws been updated? This is the way I would do it.

**Reply by mxav1111** [reply]
Thank you.  Please can you give more details on command - file - string to look for? 

Is this how really it works? Everyone with free certs has to bounce every three months? That sounds so unreasonable :-(.
